.github/workflows/ci.yml

```diff
@@ -47,7 +47,6 @@ jobs:
       run: |
         DCO_VERBOSITY=-q script/validate/dco
         GO111MODULE=on script/setup/install-dev-tools
```

Comment:
I'm wondering, do we still need `GO111MODULE`? 🤔

Reply:
Yeah in a follow-up we should remove the ci workflow. In a follow-up will take a look at the DCO check but DCO bot might be enough?

Reply:
Yeah, I think the DCO bot should take care of that...

Reply:
+1 for remove `GO111MODULE`